test/parallel/test-crypto-fips.js

```diff
@@ -19,17 +19,13 @@ const FIPS_ERROR_STRING =
 const FIPS_ERROR_STRING2 =
   'Error [ERR_CRYPTO_FIPS_FORCED]: Cannot set FIPS mode, it was forced with ' +
   '--force-fips at startup.';
-const OPTION_ERROR_STRING = 'bad option';
+const FIPS_UNSUPPORTED_ERROR_STRING = 'fips mode not supported';
```

Comment:
On one hand, yes, it could be recycled. On the other hand, the naming convention of `FIPS_ERROR_STRING{n}` is not exactly self-documenting :wink: Overall, I would prefer to keep the new name.